packages/devtools_app/lib/src/app.dart

```diff
@@ -311,6 +311,10 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
       ),
       routerDelegate: DevToolsRouterDelegate(_getPage),
       routeInformationParser: DevToolsRouteInformationParser(),
+      // Disable default scrollbar behavior on web to fix duplicate scrollbars
+      // bug, see https://github.com/flutter/flutter/issues/90697:
+      scrollBehavior:
+          const MaterialScrollBehavior().copyWith(scrollbars: !kIsWeb),
```

Comment:
what does this look like on web? Are scrollbars now gone? We really only care about how web looks since that's the only platform we ship on.

Reply:
We still have the scrollbars, but the "phantom" scrollbar that was appearing in the debugger is now gone.

![scrollbars](https://user-images.githubusercontent.com/21270878/134949714-6383dcc8-06f6-44f6-ac7b-51e20e86504e.gif)

Reply:
I see another problem here that the vertical scrollbar on the debugger does not show up until after the file has been scrolled, but I think this may be an independent issue of the phantom scrollbar problem.

Reply:
Oh interesting! I think this might be intended, we have `isAlwaysShown` set to `false`. From the [definition](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/scrollbar.dart#L717-L721):

> By default, the thumb will fade in and out as the child scroll view scrolls. When [isAlwaysShown] is true, the scrollbar thumb will remain visible without the fade animation. 

I'll open a separate PR to set `isAlwaysShown` to true.